src/core/agent/services/multiSigService.ts

```diff
@@ -530,6 +530,17 @@ class MultiSigService extends AgentService {
     return { done: false };
   }
 
+  @OnlineOnly
```

Comment:
actually can we remove this? I think it's best if we just have `@Online` in functions that directly call Signify. `rotateIdentifier` has the `@OnlineOnly`. Otherwise its very hard to track which functions call other functions that use Signify.

And we have tests for onlineonly usage